packages/sdks/snippets/react/src/main.tsx

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom/client';
 import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 import AnnouncementBar from './routes/AnnouncementBar.tsx';
 import BlogArticle from './routes/blueprints/BlogArticle.tsx';
+import { NavLinks } from './routes/blueprints/NavLinks.tsx';
```

Comment:
this is a component and not a route, should we place it inside components/ dir?

Reply:
my bad, i thought we were importing the NavBar component here. Could we make the Navlinks default exported like the others?